webapp/sources/pom.xml

```diff
@@ -422,7 +422,7 @@ limitations under the License.
     <rudder-major-version>8.2</rudder-major-version>
     <rudder-version>8.2.0~alpha1-SNAPSHOT</rudder-version>
 
-    <scala-version>2.13.12</scala-version>
+    <scala-version>2.13.13</scala-version>
```

Comment:
we should not have any issue since we use `-Xsource:3-cross` when migrating to scala 3